stratum/hal/lib/barefoot/bf_pal_wrapper.h

```diff
@@ -51,9 +51,12 @@ class BFPalWrapper : public BFPalInterface {
 
   bool PortIsValid(int unit, uint32 port_id) override;
 
-  ::util::Status PortLoopbackModeSet(int uint, uint32 port_id,
+  ::util::Status PortLoopbackModeSet(int unit, uint32 port_id,
                                      LoopbackState loopback_mode) override;
 
+  ::util::Status PortIdFromPortKeyGet(int unit, const PortKey& port_key,
```

Comment:
This function should have signature:
```
::util::StatusOr<uint32> PortIdFromPortKeyGet(
      int unit, const PortKey& port_key)
```

> Prefer using return values over output parameters: they improve readability, and often provide the same or better performance.
> Use non-const pointers to represent **optional** outputs.

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs